<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CuteEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Hilfe" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about" />
   </widget>
   <widget class="QMenu" name="menu_Bearbeiten" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_undo" />
    <addaction name="action_redo" />
    <addaction name="separator" />
    <addaction name="action_cut" />
    <addaction name="action_copy" />
    <addaction name="action_paste" />
   </widget>
   <widget class="QMenu" name="menu_Datei" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_new" />
    <addaction name="action_open" />
    <addaction name="action_save" />
    <addaction name="action_saveas" />
    <addaction name="separator" />
    <addaction name="action_quit" />
   </widget>
   <addaction name="menu_Datei" />
   <addaction name="menu_Bearbeiten" />
   <addaction name="menu_Hilfe" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="action_new" />
   <addaction name="action_open" />
   <addaction name="action_save" />
   <addaction name="separator" />
   <addaction name="action_cut" />
   <addaction name="action_copy" />
   <addaction name="action_paste" />
  </widget>
  <action name="action_open" >
   <property name="icon" >
    <iconset resource="pics.qrc" >:/pics/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="statusTip" >
    <string>Opens a file</string>
   </property>
   <property name="whatsThis" >
    <string/>
   </property>
   <property name="shortcut" >
    <string>0</string>
   </property>
  </action>
  <action name="action_saveas" >
   <property name="icon" >
    <iconset>pics/filesaveas.png</iconset>
   </property>
   <property name="text" >
    <string>Save &amp;as</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="action_quit" >
   <property name="icon" >
    <iconset>pics/fileclose.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_about" >
   <property name="text" >
    <string>&amp;Ãœber...</string>
   </property>
  </action>
  <action name="action_new" >
   <property name="icon" >
    <iconset>pics/filenew.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="shortcut" >
    <string>N</string>
   </property>
  </action>
  <action name="action_save" >
   <property name="icon" >
    <iconset>pics/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>S</string>
   </property>
  </action>
  <action name="action_undo" >
   <property name="icon" >
    <iconset>pics/undo.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_redo" >
   <property name="icon" >
    <iconset>pics/redo.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_copy" >
   <property name="icon" >
    <iconset>pics/editcopy.png</iconset>
   </property>
   <property name="text" >
    <string>C&amp;opy</string>
   </property>
  </action>
  <action name="action_paste" >
   <property name="icon" >
    <iconset>pics/editpaste.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_cut" >
   <property name="icon" >
    <iconset>pics/editcut.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Cut</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pics.qrc" />
 </resources>
 <connections/>
</ui>
